README.md

```diff
@@ -1,19 +1,126 @@
-# ASUS (UM|M)5606 fan state script
+# ASUS (UM|M)5606/5506 Fan State Control Script
 
-Bash script to set the fan state on the ZenBook S 16 UM5606, Vivobook M5606, Zenbook S 14 UX5406SA and Vivobook S 15/16 S5506
+Bash script to **view, set, and persist** the fan state on the following ASUS laptops:
+
+- **ZenBook S 16 UM5606**
```

Comment:
This is the only part of the readme changes that should stay, everything else should be reverted